README.md

```diff
@@ -4,13 +4,11 @@
 
 👋 **Welcome!**
 
-We’ve assembled a toolkit that university instructors can use to easily prepare labs, homework, or classes. The content is self-contained so that it can be easily incorporated into an existing curriculum. This content is **free** and uses well-known Open Source technologies (`transformers`, `gradio`, etc).
+We’ve assembled a toolkit that anyone can use to easily prepare workshops, events, homework or classes. The content is self-contained so that it can be easily taken over. This content is **free** and uses well-known Open Source technologies (`transformers`, `gradio`, etc).
```

Comment:
```suggestion
We’ve assembled a toolkit that anyone can use to easily prepare workshops, events, homework or classes. The content is self-contained so that it can be easily incorporated in other material. This content is **free** and uses well-known Open Source technologies (`transformers`, `gradio`, etc).
```